runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1028,9 +1028,15 @@ obj:;
 	 * @returns the new object, or NULL if allocation failed
 	 */
 	VMINLINE j9object_t
-	allocateObject(REGISTER_ARGS_LIST, J9Class *clazz, bool initializeSlots = true, bool memoryBarrier = true)
+	allocateObject(REGISTER_ARGS_LIST, J9Class *clazz, bool skipCheck = false, bool initializeSlots = true, bool memoryBarrier = true)
```

Comment:
can you rename `skipCheck` to `skipUnflattenedFlattenableCheck`, also move it to the last parameter